src/main/java/com/zendesk/maxwell/MaxwellAbstractRowsEvent.java

```diff
@@ -208,7 +208,6 @@ public String toSQL() {
 					getDatabase().getName(),
 					getTable().getName(),
 					getHeader().getTimestamp() / 1000,
-					getXid(),
```

Comment:
may not need https://github.com/zendesk/maxwell/pull/172/files#diff-6af601f94c7136d68fd09c12665d16a1R215 because of setTxCommit below when we encounter XID